resources/i18n/es_ES/cura.po

```diff
@@ -1978,17 +1963,17 @@ msgstr "Acuerdo de usuario"
 #: /home/ruben/Projects/Cura/plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobCard.qml:254
 msgctxt "@info"
 msgid "These options are not available because you are monitoring a cloud printer."
-msgstr ""
+msgstr "Estas opciones no se encuentran disponibles porque está supervisando una impresora en la nube."
```

Comment:
Note: Do not translate 'cloud' for cloud printers.

Reply:
In this case we do accept it because it's not the brand name.